templates/sections/event-themes.html

```diff
@@ -2,21 +2,24 @@
   <div class="row-fluid container">
     <div class="col-lg-6 col-md-6 col-sm-12 col-xs-12 regular-prose semibold">
       <p>
-        Meta Refresh is a conference on the construction of the user experience on the web. Meta Refresh is a conference for working web designers and developers to present on how their own work has advanced the state of the web in the past year. You are expected to present original work that your peers — this event’s audience — recognise as being notable enough to deserve a stage.
+        Meta Refresh is a conference on user experience for the web. Speakers in this event are working web designers and developers who describe how their work has advanced the state of the web in the past year. As a speaker, you are expected to present original work that your peers — the event’s audience — recognise as being notable enough to deserve a stage.
       </p>
       <p>
-        For 2015 our focus was on (a) Design from a web engineering standpoint, and (b) Enhancing the state of web experiences on mobile devices, with their many constraints.
+        In 2015, the event focussed on (a) Design from a web engineering standpoint, and (b) Enhancing the state of web experiences on mobile devices with their many constraints.
       </p>
     </div>
     <div class="col-lg-6 col-md-6 col-sm-12 col-xs-12 regular-prose semibold">
       <p>
-        We’re already in a world where smartphones outnumber all the desktops and laptops put . Wearables – smart watches and devices – now act as remote controls for notifications on our.
+        Today, we’re in a world where smartphones outnumber desktops and laptops put together. It is emperical for nearly every business with an online presence today to have a smooth mobile experience. 
```

Comment:
> It is emperical for nearly every business with an online presence today to have a smooth mobile experience.

Did you mean 'imperative' instead of 'emperical'?